From 42acfe9c3bb27da3fd98f676057e919082a59e96 Mon Sep 17 00:00:00 2001
From: Alison Wang <alison.wang@nxp.com>
Date: Tue, 26 Feb 2019 09:07:59 +0800
Subject: [PATCH 653/706] arm64: dts: ls1028a: Correct ARM PMU interrupt number

This patch corrects ARM PMU interrupt number for LS1028A.

Signed-off-by: Alison Wang <alison.wang@nxp.com>
(cherry picked from commit 76b08024efbd3ed6046e89b5550990bdd28de1a0)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 7 ++-----
 1 file changed, 2 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index eefa52d4e2da..0b72187e7060 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -168,11 +168,8 @@
 
 
 	pmu {
-		compatible = "arm,cortex-a72-pmu";
-		interrupts = <GIC_SPI 38 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 39 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 40 IRQ_TYPE_LEVEL_HIGH>,
-			     <GIC_SPI 41 IRQ_TYPE_LEVEL_HIGH>;
+		compatible = "arm,armv8-pmuv3";
+		interrupts = <GIC_PPI 7 IRQ_TYPE_LEVEL_HIGH>;
 	};
 
 	gic: interrupt-controller@6000000 {
-- 
2.17.1

